Core/UniquePixel.swift

```diff
@@ -52,14 +52,15 @@ public final class UniquePixel {
     /// This requires the pixel name to end with `_u`
     public static func fire(pixel: Pixel.Event,
                             withAdditionalParameters params: [String: String] = [:],
+                            includedParameters: [Pixel.QueryParameters] = [.appVersion],
```

Comment:
Added this to the UniquePixel API so that the caller can control the parameters, before this it was relying on the default parameters.